source/ios/AdaptiveCards/ADCIOSFramework/ADCIOSFramework/ACRActionDelegate.h

```diff
@@ -0,0 +1,15 @@
+//
+//  ACRSubmitActionDelegate
+//  ACRSubmitActionDelegate.h
+//
+//  Copyright © 2017 Microsoft. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+
+@protocol ACRActionDelegate
+
+- (void)fetchUserResponses:(NSDictionary *)userResponses;
```

Comment:
thanks

---
In reply to: [132051951](https://github.com/Microsoft/AdaptiveCards/pull/532#discussion_r132051951) [](ancestors = 132051951)